frontend/model/contracts/group.js

```diff
@@ -161,6 +164,21 @@ DefineContract({
         }
       }
     },
+    'gi.contracts/group/proposalCancel': {
```

Comment:
There already exists a `'gi.contracts/group/proposalWithdraw'` and that is what should be used/updated instead